resources/ssm_instances.go

```diff
@@ -0,0 +1,267 @@
+package resources
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	"github.com/aws/aws-sdk-go-v2/service/ssm/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func SsmInstances() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_ssm_instances",
+		Description:  "Describes a filter for a specific list of instances.",
+		Resolver:     fetchSsmInstances,
+		Multiplex:    client.AccountRegionMultiplex,
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountRegionFilter,
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"account_id", "region", "instance_id"}},
+		Columns: []schema.Column{
```

Comment:
can we add ARN for the resource https://docs.aws.amazon.com/service-authorization/latest/reference/reference_policies_actions-resources-contextkeys.html if it exists ?

Reply:
Added